dts/arm/nxp/nxp_k8x.dtsi

```diff
@@ -351,7 +351,7 @@
 			compatible = "nxp,kinetis-dspi";
 			reg = <0x400ac000 0x1000>;
 			interrupts = <65 3>;
-			clocks = <&sim KINETIS_SIM_BUS_CLK 0x1030 12>;
+			clocks = <&sim KINETIS_SIM_BUS_CLK 0x103c 23>;
```

Comment:
Why are you changing a SPI clock gate in a PIT commit? Is this a mistake?

Reply:
nice shot, this should change to the pit0, I wrongly put it to dspi, although we do not access the clock gate register bit in code.